security/pkg/monitoring/monitoring.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitoring
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	configID = "configID"
+)
+
+var (
+	caLabels = []string{configID}
+
+	successCertIssuanceCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "citadel",
+		Subsystem: "ca",
+		Name:      "success_cert_issuance_count",
+		Help:      "The number of certificates issuances that have succeeded.",
+	}, caLabels)
+
+	cSRErrorCount = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
We should use csrErrorCount, the abbreviation should be all little case or capital case.
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Same for the rest of variable.